packages/integration-tests/migrations/1580658856631_author.ts

```diff
@@ -198,6 +201,8 @@ export function up(b: MigrationBuilder): void {
     parent_book_review_id: foreignKey("book_reviews", { notNull: false, unique: true }),
     parent_publisher_id: foreignKey("publishers", { notNull: false }),
     parent_author_id: foreignKey("authors", { notNull: false }),
+    // for testing collection renames
+    user_id: foreignKey("users", { notNull: true, collectionName: "createdComments" }),
```

Comment:
Nit: Maybe make this `notNull: false`? Would reduce the tests that need to create users just to use comments for their boundary cases.

Reply:
Should we really allow anonymous commenting? seems like a slippery slope towards becoming 4chan 😋